src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml

```diff
@@ -326,7 +326,7 @@
                                  ItemsSource="{Binding MonitorInfoForViewModel}" />
                 </Grid>
             </ScrollViewer>
-            <Button Click="SettingsBtn_Click"
+            <!--<Button Click="SettingsBtn_Click"
```

Comment:
Please add a comment like `Commented because of issue http://.... (crutkas). ToDo: Fix window order bug under Win11 for settings window.` that we know later why it is commented.

Reply:
Agreed, a comment would be useful here.